.all-contributorsrc

```diff
@@ -266,6 +266,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
```

Comment:
Sorry, but remove this part, because I'll add you as a contributor.

I always do this when I release.

Reply:
Resolvido!